CHANGES.rst

```diff
@@ -4,7 +4,12 @@ Changes
 Version 0.8.1 (UNRELEASED)
 --------------------------
 
+- Users:
+    - Adds support for specifying ``kubernetes_job_timeout`` for Kubernetes compute backend jobs for CWL, Serial, Snakemake, and Yadage workflows.
```

Comment:
Since the feature is available in all the workflow engines, perhaps we don't need to mention them?